CHANGELOG/CHANGELOG-2.12.md

```diff
@@ -0,0 +1,106 @@
+# Release notes for v2.12.0
+
+[Documentation](https://kubernetes-csi.github.io/docs/)
+
+# Changelog since v2.11.0
+
+
+## Changes by Kind
+
+### Feature
+
+- Added support for structured logging (the log messages have been changed due to the activation of structured logging) ([#202](https://github.com/kubernetes-csi/livenessprobe/pull/202), [@bells17](https://github.com/bells17))
+
+### Bug or Regression
+
+- Liveness probe process does not crash when it cannot access the associated CSI driver. It only fails all kubelet probes, most probably with "connection refused". ([#240](https://github.com/kubernetes-csi/livenessprobe/pull/240), [@jsafrane](https://github.com/jsafrane))
+
+### Uncategorized
+
+- CVE fixes: CVE-2023-44487 ([#220](https://github.com/kubernetes-csi/livenessprobe/pull/220), [@dobsonj](https://github.com/dobsonj))
+- Update kubernetes dependencies to v1.29.0 ([#238](https://github.com/kubernetes-csi/livenessprobe/pull/238), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
+- Update kubernetes dependencies to v1.29.0-rc.1 ([#230](https://github.com/kubernetes-csi/livenessprobe/pull/230), [@sunnylovestiramisu](https://github.com/sunnylovestiramisu))
```

Comment:
Can you remove this one as this is a RC version?

Reply:
Removed